test/CourseInfo_test.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file CourseInfo_test.cpp
+ * @brief コース情報のテスト
+ * @author higuchi, miyashita
+ */
+
+#include "CourseInfo.h"
+
+#include <gtest/gtest.h>
+
+namespace etrobocon2021_test {
+
+  // CourseInfoコンストラクタに関するテスト
+  // getBrightness()に関するテスト
+  // 輝度を返す処理
+  TEST(getTargetBrightnessTest, getTargetBrightness)
+  {
+    CourseInfo info(1);
+    int expected = 1;
+
+    EXPECT_EQ(expected, info.getTargetBrightness());
+  }
+
+  // 輝度0を返す処理
+  TEST(getTargetBrightnessTest, getTargetBrightnessByZeroValue)
+  {
+    CourseInfo info(0);
+    int expected = 0;
+
+    EXPECT_EQ(expected, info.getTargetBrightness());
+  }
+  //輝度100を返す処理
+  TEST(getTargetBrightnessTest, getTargetBrightnessByOneHundredValue)
+  {
+    CourseInfo info(0);
+    int expected = 0;
```

Comment:
```suggestion
    CourseInfo info(100);
    int expected = 100;
```
輝度0を返す処理と同じことが書いてあるので、輝度100を返す処理に書き換えたほうがいいと思います